### PR TITLE
remove v1 vs v2 mentions

### PR DESCRIPTION
**Pull request by rodrigonogueira4**
- Remove mentions to v1 vs v2 in the download table
- Add MRR@10 to the to table to facilitate which model to choose from.
- Remove mention to cost-benefit paper.

### CODE REVIEW

**Comment by rodrigonogueira4** [comment]
@lhbonifacio @hugoabonizio please let me know if you have any suggestions regarding the presentation. I'm happy to change it in case you feel so.

File: README_old.md [hunk 1]
```diff
@@ -157,7 +99,7 @@ QueriesRanked: 6980
 ## Re-ranking with mT5
```
Comment (hugoabonizio) [inline README_old.md:99]:
```suggestion
## Re-ranking with PTT5
```

File: README_old.md [hunk 2]
```diff
@@ -157,7 +99,7 @@ QueriesRanked: 6980
...
 Finally, we can re-rank our BM25 initial run using [mT5-base-multi-msmarco](https://huggingface.co/unicamp-dl/mt5-base-multi-msmarco) (or each one of the previous listed models):
```
Comment (hugoabonizio) [inline README_old.md:100]:
```suggestion
Finally, we can re-rank our BM25 initial run using [ptt5-base-en-pt-msmarco-10k](https://huggingface.co/unicamp-dl/ptt5-base-en-pt-msmarco-10k) (or each one of the previous listed models):
```

File: README_old.md [hunk 3]
```diff
@@ -1,75 +1,20 @@
...
--->
-
-## Translated Datasets
-We translate MS MARCO passage ranking dataset, a large-scale IR dataset comprising more than half million anonymized questions that were sampled from Bing's search query logs. There are two translated versions of mMARCO.
 
-  * **Helsinki**  
-  In v1 version, we use MarianNMT an open-source neural machine translation framework originally written in C++ for fast training and translation. The Language Technology Research Group at the University of Helsinki made available [more than a thousand language pairs](https://huggingface.co/Helsinki-NLP) for translation, supported by HuggingFace framework. The code [available here](https://github.com/unicamp-dl/mMARCO/blob/main/scripts/translate.py) is the same used in our paper.
-  This version comprises 8 languages: Chinese, French, German, Indonesian, Italian, Portuguese, Russian and Spanish.
-  In the paper, we refer to these models as "Helsinki".
+** This README refers to the old version (v1) of mMARCO. We highly recommend you use the latest dataset and models from the main [README.d] **
```
Comment (hugoabonizio) [inline README_old.md:3]:
```suggestion
**This README refers to the old version (v1) of mMARCO. We highly recommend you use the latest dataset and models from the main [README.md](/README.md)**
```

**Review by hugoabonizio — APPROVED** [review]
Only a few suggestions, but overall it's very nice :+1:

**Comment by lhbonifacio** [comment]
Thanks @hugoabonizio for the minor changes!
LGTM!

**Pull request merged**